client/views/admin/info/InformationRoute.tsx

```diff
@@ -1,30 +1,37 @@
 import { Callout, ButtonGroup, Button, Icon } from '@rocket.chat/fuselage';
-import React, { useState, useEffect, memo } from 'react';
+import React, { useState, useEffect, memo, ReactElement } from 'react';
 
+import { IServerInfo } from '../../../../definition/IServerInfo';
+import { IStats } from '../../../../definition/IStats';
 import NotAuthorizedPage from '../../../components/NotAuthorizedPage';
 import Page from '../../../components/Page';
+import PageSkeleton from '../../../components/PageSkeleton';
 import { usePermission } from '../../../contexts/AuthorizationContext';
 import { useMethod, useServerInformation, useEndpoint } from '../../../contexts/ServerContext';
 import { useTranslation } from '../../../contexts/TranslationContext';
 import { downloadJsonAs } from '../../../lib/download';
-import NewInformationPage from './NewInformationPage';
+import InformationPage from './InformationPage';
 
-const InformationRoute = memo(function InformationRoute() {
+type fetchStatisticsCallback = ((params: { refresh: boolean }) => void) | (() => void);
+
+const InformationRoute = (): ReactElement => {
 	const t = useTranslation();
 	const canViewStatistics = usePermission('view-statistics');
 
 	const [isLoading, setLoading] = useState(true);
-	const [error, setError] = useState();
-	const [statistics, setStatistics] = useState({});
+	const [error, setError] = useState<boolean>(false);
```

Comment:
It must enough to infer from the initial value.

```suggestion
	const [error, setError] = useState(false);
```